gears/src/baseapp/abci.rs

```diff
@@ -63,8 +63,9 @@ impl<DB: Database, PSK: ParamsSubspaceKey, H: ABCIHandler, AI: ApplicationInfo>
 
         let mut ctx = state.init_ctx(initial_height, time, chain_id);
 
+        // TODO: do we care about error in the consensus params?
         self.baseapp_params_keeper
-            .set_consensus_params(&mut ctx, consensus_params.clone().into());
+            .set_consensus_params(&mut ctx, consensus_params.clone().try_into().unwrap());
```

Comment:
I've changed our tendermint type. I replaced `Option<Timestamp>` to `Timestamp` and then I used `TryFrom` instead of `From`.  
The sdk has no option for `time` field in the Evidence type. But I can rollback the changes and it became infallible again.